ansible/README.md

```diff
@@ -0,0 +1,3 @@
+# trento/ansible
```

Comment:
Most probably we need to upgrade this documentation.
For other occasion I guess